### PR TITLE
MCL1, SHP2, Renin, HNE sets added. 

### PR DESCRIPTION
**Pull request by alexanderhwilliams**
## Input submission checklist

* [X] created a directory for each system based on the [template directory][templates]
* [X] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [X] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [X] added a PDB file with the protein named `protein.pdb`
* [X] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [X] did not re-protonate the residues, but left protonation state of the protein unchanged
* [X] kept cystallographic waters and metals in the PDB file
* [X] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [X] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb:1]: Is it possible that accidentally the wrong file was deposited here? This seems to be the HNE protein, very similar to the HNE protein deposited in the hne folder, only that the two versions have slightly different positions of protons.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/cofactor.sdf
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/cofactor.sdf:1]: Similar to the comment above, potentially remove the NAG molecule.

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/mcl1/PREPARATION_DETAILS.md
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/jacs_set/mcl1/PREPARATION_DETAILS.md:1]: Thank you, this looks great!

**Review by hannahbaumann — CHANGES_REQUESTED** [review]

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/PREPARATION_DETAILS.md
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/PREPARATION_DETAILS.md:13]: This is a glycosylation site - i.e. bound to the nearby ASN. Given it's far from the binding site and we won't be able to handle the glycosylation properly, we probably just should remove it and revert to a normal ASN.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb:1]: Just echo-ing Hannah's comment that this seems to be the wrong protein.

**Review by IAlibay — CHANGES_REQUESTED** [review]
Couple of extra things to add on top of @hannahbaumann's review comments.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb:1]: Yes, wrong protein was placed in this folder, will be removing in next commit.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/ligands.sdf
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/ligands.sdf:1]: Errant Ligands 21,19,16 in neutral form have been removed.

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/mcl1/PREPARATION_DETAILS.md
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/jacs_set/mcl1/PREPARATION_DETAILS.md:10]: Clarifying this in next commit.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/PREPARATION_DETAILS.md
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/PREPARATION_DETAILS.md:10]: ACE Cap has been added to ILE-16 in next commit.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/protein.pdb
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/protein.pdb:1]: I noticed that both termini (Ile16 and Gln243) were charged in the original Schrodinger structure, For this benchmarking study we decided to leave termini in their charged state and not to cap them to stay consistent with the Schrodinger preparation (https://industrybenchmarks2024.readthedocs.io/en/latest/public/input_preparation.html#capping-proteins). Could you revert the CAPs back to the charged state of the amino acid?

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/PREPARATION_DETAILS.md
Comment (hannahbaumann) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/PREPARATION_DETAILS.md:13]: Thank you for removing the NAG cofactor! Could you update the preparation details to reflect those changes? Thank you!

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb:1]: Thank you for updating the file, this looks good! Only thing I noticed is that the terminus Arg326 was charged in the original structure. Could you remove the NME cap here?

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thank you @alexanderhwilliams ! My apologies for the delay in reviewing! Thank you for addressing the comments! I left some small comments, please let me know if you have any questions!

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/cofactors.sdf
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/cofactors.sdf:1]: FUC1 and FUC2 have been removed from cofactors.sdf

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/protein.pdb
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/protein.pdb:1]: Termini are now for HNE are uncapped, this has been reflected in the preparation details.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb:1]: Arg326 has been decapped to its original uncharged state, prep details have been updated to reflect this.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/PREPARATION_DETAILS.md
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/PREPARATION_DETAILS.md:13]: Updated preparation details.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/cofactor.sdf
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/cofactor.sdf:1]: Just a tiny comment, for consistency with the file structure, I think we still want to name this `cofactors.sdf` even though there is just a single cofactor in it. But we can also adapt that if necessary afterwards!

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/PREPARATION_DETAILS.md
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/PREPARATION_DETAILS.md:20]: I noticed that you had removed the neutral forms of the ligands 16, 19, 21 as per the prep instructions, could you add this information here?

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb:5123]:
```suggestion
ATOM   5119  OXT ARG A 326      -2.057   6.763 -21.573  1.00  0.00           O1-
```
I think we need to rename the terminal oxygen to OXT to follow PDB conventions, otherwise the run script was failing for me.

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thank you @alexanderhwilliams for addressing the comments! I just added a small comment and a suggestion!

File: .DS_Store
Comment (hannahbaumann) [inline .DS_Store:1]: Would it be ok if we delete this and the other `.DS_Store` files before merging?

Comment (alexanderhwilliams) [reply .DS_Store:1]: Hi Hannah, my apologies about these, they apparently get added with the Github desktop client. I have removed them now.

Comment (alexanderhwilliams) [reply .DS_Store:1]: Files have been removed.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/cofactor.sdf
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/cofactor.sdf:1]: This has been corrected.

File: industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/PREPARATION_DETAILS.md
Comment (alexanderhwilliams) [reply industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/hne/PREPARATION_DETAILS.md:20]: Updated preparation details to reflect this.

File: industry_benchmarks/input_structures/prepared_structures/merck/shp2/ligands.sdf
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/merck/shp2/ligands.sdf:1]:
```suggestion
SHP099-1 Example 7
```
The `/` is going to cause some issues at simulation time (since it'll get added to the file name).

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/mcl1/protein.pdb
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/jacs_set/mcl1/protein.pdb:1]: The GLY termini was uncapped in the original schrodinger structure, it should be here too.

**Review by IAlibay — COMMENTED** [review]
This is nearly there but there are two things that need addressing - please see my email regarding this.

**Comment by IAlibay** [comment]
@alexanderhwilliams - we're going to go ahead and merge this PR, but will follow up with another to fix up the mcl1 gly cap and the hne ligand SDF format.

Please wait until we merge the follow-up PR before starting the simulations, and then you'll need to pull down the inputs held on the `main` branch of this repository (rather than this one).

**Pull request merged**
